packages/cli/package.json

```diff
@@ -45,7 +45,7 @@
     "test": "NODE_OPTIONS=\"--experimental-vm-modules\" vitest"
   },
   "dependencies": {
-    "@ts-bridge/resolver": "workspace:^",
+    "@ts-bridge/resolver": "^0.1.1",
```

Comment:
This makes testing the package easier, as `yarn link` doesn't support `workspace:^`.